experimental/sdg-tracker/src/components/goals/Goals.tsx

```diff
@@ -65,7 +66,7 @@ const Goals: React.FC = () => {
               hidePlaceSearch={true}
               variableDcids={[variableDcid]}
               placeDcids={[EARTH_PLACE_DCID]}
-              setPlaceDcid={() => {}}
+              setPlaceDcid={() => { }}
```

Comment:
Can you install the vscode "prettier" plugin for code formatting (and enable "run on save") - that will keep code formatting like here consistent across everyone
https://www.robinwieruch.de/how-to-use-prettier-vscode/

Reply:
Done.